bindings/c/test/mako/mako.c

```diff
@@ -212,17 +218,40 @@ int populate(FDBTransaction *transaction, mako_args_t *args, int worker_id,
         xacts = 0;
         timer_prev.tv_sec = timer_now.tv_sec;
         timer_prev.tv_nsec = timer_now.tv_nsec;
+
+	/* enable transaction tracing */
+	if (args->txntrace) {
+	  tracetimer++;
+	  if (tracetimer == args->txntrace) {
+	    fdb_error_t err;
+	    tracetimer = 0;
+	    fprintf(debugme, "DEBUG: txn tracing %s\n", keystr);
+	    err = fdb_transaction_set_option(transaction,
+					     FDB_TR_OPTION_DEBUG_TRANSACTION_IDENTIFIER,
+					     (uint8_t *)keystr, strlen(keystr));
+	    if (err) {
+	      fprintf(stderr,
+		      "ERROR: fdb_transaction_set_option(FDB_TR_OPTION_DEBUG_TRANSACTION_IDENTIFIER): %s\n",
+		      fdb_get_error(err));
+	    }
+	    fdb_transaction_set_option(transaction,
+				       FDB_TR_OPTION_LOG_TRANSACTION,
+				       (uint8_t *)NULL, 0);
+	    if (err) {
+	      fprintf(stderr,
+		      "ERROR: fdb_transaction_set_option(FDB_TR_OPTION_DEBUG_TRANSACTION_IDENTIFIER): %s\n",
```

Comment:
```suggestion
		      "ERROR: fdb_transaction_set_option(FDB_TR_OPTION_LOG_TRANSACTION): %s\n",
```

Reply:
Yes, I was aware of this issue.  This file was originally and still is written without clang-format.  Once this is merged, I'll make another PR to cleanup indentation.

Reply:
Looking more closely, I see that `err` isn't being assigned for the second call to `fdb_transaction_set_option`. Is it supposed to be? My change to the message text was assuming that it was.